lib/govuk_publishing_components/presenters/attachment_helper.rb

```diff
@@ -1,7 +1,7 @@
 module GovukPublishingComponents
   module Presenters
     class AttachmentHelper
-      delegate :opendocument?, :document?, :spreadsheet?, :pdf?, to: :content_type
+      delegate :opendocument?, :document?, :spreadsheet?, :pdf?, :external?, to: :content_type
```

Comment:
I'm a bit confused here because we delegate external? to the content_type, but then provide an implementation of it.

Reply:
Sorry, that was left over from some experimenting I was doing to get this to work. Have removed.